--quiet disables --fix

Invoking directly eslint with `--fix --quiet` fixes the detected autofixable non-warning errors (as per eslint documentation https://eslint.org/docs/user-guide/command-line-interface#-quiet)

However, `eslint_d --fix --quiet` does not fix any error and reports to use --fix, even when it's already included:

```
/eslint_d  --quiet --fix src/cms/callback.ts

src/cms/callback.ts
  3:10  error  Delete `···`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

```

--- a/lib/linter.js
+++ b/lib/linter.js
@@ -49,8 +49,7 @@ function translateOptionsESLint(cliOptions, cwd) {
     cacheLocation: cliOptions.cacheLocation || cliOptions.cacheFile,
     errorOnUnmatchedPattern: cliOptions.errorOnUnmatchedPattern,
     extensions: cliOptions.ext,
-    fix: (cliOptions.fix || cliOptions.fixDryRun || cliOptions.fixToStdout)
-      && (cliOptions.quiet ? cliOptions.quietFixPredicate : true),
+    fix: cliOptions.fix || cliOptions.fixDryRun || cliOptions.fixToStdout,
     fixTypes: cliOptions.fixType,
     ignore: cliOptions.ignore,
     ignorePath: cliOptions.ignorePath,

--- a/test/linter-test.js
+++ b/test/linter-test.js
@@ -224,6 +224,13 @@ describe('linter', () => {
           assert.calledOnceWith(callback, null, 'console.log(\'!\');\n');
         });
 
+        it('returns fixed script also with --quiet', async () => {
+          await linter.invoke(dir, ['--stdin', '--fix-to-stdout', '--quiet'],
+            'console.log( "!" )\n', 0, callback);
+
+          assert.calledOnceWith(callback, null, 'console.log(\'!\');\n');
+        });
+
         it('fails if --stdin is not given', async () => {
           await linter.invoke(dir, ['--fix-to-stdout', '.'], '', 0, callback);
 

